src/app/space/create/deployments/services/deployments.service.ts

```diff
@@ -133,8 +133,11 @@ export class DeploymentsService {
 
   getEnvironments(spaceId: string): Observable<ModelEnvironment[]> {
     return this.getEnvironmentsResponse(spaceId)
-      .map((envs: EnvironmentStat[]) => envs || [])
-      .map((envs: EnvironmentStat[]) => envs.map((env: EnvironmentStat) => ({ name: env.name} as ModelEnvironment)))
+      .map((envs: EnvironmentStat[]) => envs ? envs.sort((a, b) =>  -1 * a.name.localeCompare(b.name)) : [])
```

Comment:
I think it might be slightly more readable if the `envs || []` mapping was left on its own line, and then afterward the mapping applying the sort function could be done without having to nullcheck the envs parameter.